from sys import argv

print(argv[0])
print(argv)
print(argv[1])
'''
When we run the program as python3 prg5.py
At index zero prg5.py is stored
and nothing is at index 1
hence index error at line 5
'''
